frontend/pages/[gameId]/[playerId].tsx

```diff
@@ -0,0 +1,57 @@
+import { WEBSOCKET_BASE_URL } from '/Users/gealoro/kakaw/frontend/lib/api';
```

Comment:
```suggestion
import { WEBSOCKET_BASE_URL } from '@/lib/api';
```

The current import only works for you. `@` refers to the root of the package (in this case `frontend`).